tools/generator/autorest/model/changelog.go

```diff
@@ -9,25 +9,39 @@ import (
 	"github.com/Azure/azure-sdk-for-go/tools/apidiff/report"
 )
 
-// Changelog describes a changelog of the package during this generation
+// Changelog describes the changelog generated for a package.
 type Changelog struct {
-	PackageName    string
-	NewPackage     bool
+	// NewPackage is true if this package does not exist in the old version
+	NewPackage bool
+	// RemovedPackage is true if this package does not exist in the new version
 	RemovedPackage bool
-	Modified       *report.Package
+	// Modified contains the details of a modified package. This is nil when either NewPackage or RemovedPackage is true
+	Modified *report.Package
```

Comment:
Go fmt?

Reply:
gofmt only requires the type to align when two lines are directly adjacent to each other

Reply:
So if there's a line of comment it doesn't work?.......